tests/unittest/auto_deploy/singlegpu/custom_ops/quantization/test_quant.py

```
@@ -147,11 +147,12 @@ def test_fp8_bmm(input_dtype, mat2_dtype):
     assert cos_sim_unquantized > 0.99
 
 
-@pytest.mark.parametrize("bias", [torch.rand(32, device="cuda") * 10, None])
+@pytest.mark.parametrize("use_bias", [True, False])
 @pytest.mark.skipif(not fp8_compatible(), reason="Requires fp8 support")
-def test_quant_linear_fp8_matches_fused_op(bias):
+def test_quant_linear_fp8_matches_fused_op(use_bias):
     input = torch.rand(3, 16, device="cuda")
```

Ruff (0.16.1)

[error] Variable `input` is shadowing a Python builtin

(A001)